src/components/QrCode/index.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { CSSTransitionGroup } from "react-transition-group";
+
+import Modal from '../Modal';
+import QrCodeSVG from '../../assets/images/qr-code.svg';
+import QrScanner from '../../libraries/qr-scanner';
```

Comment:
note for Jacky: qr-scanner doesn't have an npm module so I just imported 2 files